tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "outDir": "dist",
     "allowUnusedLabels": false,
     "noImplicitReturns": true,
+    "strictNullChecks": true,
```

Comment:
You can try to even set "strict": true," if it doesn't result in to many errors. It's always a trade-off between spending too much time with stupid code checkers and actually having helpful suggestions that discover real bugs. In general, I had the feeling typescripts strict mode was quite helpful.